pom.xml

```diff
@@ -1541,9 +1542,7 @@
                     </execution>
                 </executions>
             </plugin>
-        </plugins>
-        <pluginManagement>
-            <plugins>
+
```

Comment:
More specifically, I run mvn eclipse:eclipse and then import the project in Eclipse instead of expecting Eclipse to be able to understand the pom.xml through a plugin.